earth_enterprise/src/fusion/autoingest/tools/geaddtoimageryproject.cpp

```diff
@@ -47,7 +47,13 @@ usage(const std::string &progn, const char *msg = 0, ...)
           "      --no_historical_imagery : make this a normal project\n"
           "                         (i.e., not a historical imagery project)\n"
           "   By default, new projects are NOT time machine projects"
-          " unless --historical_imagery is specified.\n",
+          " unless --historical_imagery is specified.\n"
+          "      --terrain_overlay : make this terrain project an overlay project\n"
+          "      --no_terrain_overlay : make this terrain project a normal project\n"
```

Comment:
we may not need to add `--no_terrain-overlay` but document it as this is the default behavior. Two reasons:
1- Backward compatibility of any tool or scripts using this CLI: no need for them to change how they call the command
2- Simpler: if one need overlay terrain they add it. Otherwise nothing need to be specified.

Reply:
On the other hand, this mirrors `--historical-imagery`/`--no-historical-imagery` that already exist.